membership-common/src/main/scala/com/gu/monitoring/SafeLogger.scala

```diff
@@ -1,21 +1,33 @@
 package com.gu.monitoring
 
-import com.typesafe.scalalogging.StrictLogging
-import org.slf4j.{Marker, MarkerFactory}
+import com.gu.monitoring.SafeLogger.LogMessage
+import com.typesafe.scalalogging.Logger
+import org.slf4j.{LoggerFactory, Marker, MarkerFactory}
 
-object SafeLogger extends StrictLogging {
+trait SafeLogging {
```

Comment:
these are the changes to get the new syntax, this is just taken from support-frontend
https://github.com/guardian/support-frontend/blob/main/support-config/src/main/scala/com/gu/monitoring/SafeLogger.scala